libr/core/cbin.c

```diff
@@ -3649,52 +3649,57 @@ static void bin_pe_versioninfo(RCore *r, int mode) {
 }
 
 static void bin_elf_versioninfo(RCore *r, int mode) {
-	const char *format = "bin/cur/info/versioninfo/%s%d";
+	const char *const format = "bin/cur/info/versioninfo/%s%d";
 	int num_versym;
 	int num_verneed = 0;
 	int num_version = 0;
 	Sdb *sdb = NULL;
 	const char *oValue = NULL;
-	bool firstit_for_versym = true;
+	PJ *pj;
 	if (IS_MODE_JSON (mode)) {
-		r_cons_printf ("{\"versym\":[");
+		pj = pj_new ();
```

Comment:
Check for null

Reply:
Simply returns if null, because other calls to pj_new either don't check or just do so.